.devcontainer/devcontainer.json

```diff
@@ -13,17 +13,17 @@
 		"python.linting.enabled": true,
 		"python.linting.flake8Enabled": true,
 		"python.formatting.provider":"black",
-		"python.testing.pytestEnabled":true,
+		"python.testing.pytestEnabled":true
 	},
 
 	"extensions": [
-		"ms-python.python",
+		"ms-python.python"
 	],
-
 	"features": {
 		"git": {
-			"version": "os-provided",
-		},
+			"version": "os-provided"
+		}
 	},
+	"postCreateCommand": "git config core.hooksPath .githooks"
```

Comment:
> Are .githooks automatically symlinked by the devcontainer?

Yes It is this line. 
For who is using the manual setup we need to put the same command in the markdown section we have at:

https://github.com/keras-team/keras-cv/blob/master/.github/CONTRIBUTING.md#setup-environment